### PR TITLE
Trus 1373 - Created maintaining your trust information page

### PR DESCRIPTION
**Pull request by JoshuaH01**

### CODE REVIEW

File: app/views/components/numbered_list.scala.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@*
```
Comment (jassalrichy) [inline app/views/components/numbered_list.scala.html]: Is this component used?

File: test/controllers/InformationMaintainingThisTrustControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+      status(result) mustEqual OK
+
+      contentAsString(result) mustEqual
+        view(fakeDraftId, utr)(fakeRequest, messages).toString
+
+      application.stop()
+    }
+  }
+}
```
Comment (jassalrichy) [inline test/controllers/InformationMaintainingThisTrustControllerSpec.scala]: Add a test for the redirect to "What Is The UTR" page ?

File: app/views/components/numbered_list.scala.html (hunk 1 above)
Comment (JoshuaH01) [reply app/views/components/numbered_list.scala.html]: No, it was made to be used but it wouldn't render properly with the link in the list.

Comment (JoshuaH01) [reply app/views/components/numbered_list.scala.html]: Resolved

**Pull request merged**